src/irmin-mirage/graphql/irmin_mirage_graphql.ml

```diff
@@ -4,14 +4,17 @@ module Server = struct
 
     module Http : Cohttp_lwt.S.Server
 
-    module Store : Irmin.S with type Private.Sync.endpoint = Git_mirage.endpoint
```

Comment:
I think the insertion of `Mimic` on graphql comes here. I don't understand why `irmin-mirage-graphql` decided to use `Git_mirage.endpoint`. On my side, it sure that a `Git_mirage.endpoint/Smart.Endpoint.t` must be associated to a `Mimic.ctx`. I'm not use how to solve this issue however.